updater/lib/dependabot/base_command.rb

```diff
@@ -63,7 +63,7 @@ def handle_unknown_error(err)
         "package-manager" => job.package_manager,
```

Comment:
Thinking aloud: It's a little weird that everything else extracted from `job.*` has a key of `job-*`... but this doesn't. It's probably nbd but thought I'd mention it...